pkg/app/client/client_test.go

```diff
@@ -165,7 +166,7 @@ func TestClientGetWithBody(t *testing.T) {
 	opt.Addr = "unix-test-10002"
 	opt.Network = "unix"
 	engine := route.NewEngine(opt)
-	engine.GET("/", func(c context.Context, ctx *app.RequestContext) {
+	engine.POST("/", func(c context.Context, ctx *app.RequestContext) {
```

Comment:
Why change this line?

Reply:
This func name is `TestClientGetWithBody`?
Maybe add a new test func for the added logic?